sympy/integrals/tests/test_laplace.py

```diff
@@ -439,6 +440,13 @@ def simp_hyp(expr):
         c*(a*exp(b*t) - b*exp(a*t))*exp(-t*(a + b))*Heaviside(t)/(a - b)
     assert ILT(c*s/(d**2*(s+a)**2+b**2), s, t) ==\
         c*(-a*d*sin(b*t/d) + b*cos(b*t/d))*exp(-a*t)*Heaviside(t)/(b*d**2)
+    # Test time_diff rule
+    assert ILT(s**42*f(s), s, t) ==\
+        Derivative(InverseLaplaceTransform(f(s), s, t, None), (t, 42))
```

Comment:
The test case looked quite exhaustive for picking `42` as exponent though.

Reply:
;)